### ISSUE TITLE
Encapsulated Return Type Not Resolving on Docs.Microsoft.com

### ISSUE THREAD

**Issue by scbedd** [Azure/azure-sdk-for-python#9862]
This is what it looks like on the review site:

https://review.docs.microsoft.com/en-us/python/api/azure-ai-textanalytics/azure.ai.textanalytics.textanalyticsclient?view=azure-python-preview&branch=smoke-test-preview#analyze-sentiment-inputs--language-none----kwargs-
![image](https://user-images.githubusercontent.com/45376673/74482122-c9f40800-4e68-11ea-87e1-ad9800ac3614.png)

We believe that the return type should be resolving appropriately.

[Location in Code](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py#L129)
[Rendered on From Standard Sphinx](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-ai-textanalytics/1.0.0b2/azure.ai.textanalytics.html#azure.ai.textanalytics.TextAnalyticsClient.analyze_sentiment)

**Comment by scbedd** [comment]
@kristapratico made a suggestion that we can try adjusting the code to have a space between the two models in the return type. 

Will try that on our side.

**Comment by scbedd** [comment]
That didn't help. Waiting on new docs.ms python generator.

**Comment by lmazuel** [comment]
Should be `list[~azure.ai.textanalytics.DetectLanguageResult or ~azure.ai.textanalytics.DocumentError]`, @kristapratico is creating a new issue to fix the syntax first, let's re-investigate here once the `or` is published if the code is better (nothing tells us the `or` will be enough)

**Comment by scbedd** [comment]
@lmazuel @kristapratico they shipped the new generator a last week. We should recheck the outputs here.

**Comment by kristapratico** [comment]
This is fixed now. I think the syntax fix did the trick.

https://docs.microsoft.com/en-us/python/api/azure-ai-textanalytics/azure.ai.textanalytics.textanalyticsclient?view=azure-python-preview#azure-ai-textanalytics-textanalyticsclient-analyze-sentiment

![image](https://user-images.githubusercontent.com/31998003/159387396-a590eb2d-0cc2-4be7-818b-ba67131e1aa4.png)
